src/customerCases/CustomerCases.tsx

```diff
@@ -0,0 +1,30 @@
+import { RichText } from "src/components/richText/RichText";
+import Text from "src/components/text/Text";
+import { CustomerCasePage } from "studio/lib/interfaces/specialPages";
+import { CustomerCase } from "studioShared/lib/interfaces/customerCases";
+
+import styles from "./customerCases.module.css";
+
+interface CustomerCasesProps {
+  customerCases: CustomerCasePage;
```

Comment:
I find this naming a little confusing. Looks like there are non-shared and shared cases, while `customerCases` is actually cust the properties for the page showing the shared cases right? How about naming it `customerCasesPage`?